### PR TITLE
Skip building deprecated Metrics API

### PR DESCRIPTION
**Pull request by ahadnagy**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ahadnagy** [comment]
@conda-forge-admin, please rerender

File: .ci_support/linux_64_grpc_cpp1.47.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ cxx_compiler_version:
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
 grpc_cpp:
-- '1.48'
+- '1.47'
```
Comment (lidavidm) [inline .ci_support/linux_64_grpc_cpp1.47.yaml:16]: Hmm, weird. Something might've gone off with  the re-render here?

**Comment by lidavidm** [comment]
@conda-forge-admin, please rerender

File: .ci_support/linux_64_grpc_cpp1.47.yaml (hunk 1 above)
Comment (lidavidm) [reply .ci_support/linux_64_grpc_cpp1.47.yaml:16]: @mariusvniekerk pointed out the relevant migration was rolled back so this is expected

**Pull request merged**